### ISSUE TITLE
Stanford branding and link to PURL

### ISSUE THREAD

**Issue by snydman** [sul-dlss/sul-embed#482]
Now that it is released I think the lack of Stanford (SUL) branding is problematic.  I really think we need a solution, either the rosette or something else.

Also, the link to the PURL now is overly obscured, very hard for a patron to "cite" the digital object.  

@jvine and @mejackreed can we please revisit this design decision?

**Comment by hannahfrost** [comment]
I have to agree, particularly when it comes to the PURL link. Here is an example SW item record that lacks a visible PURL link: https://searchworks.stanford.edu/view/rg197ss4320. I looked twice and can't uncover it, wherever it may be obscured.

Given the primary role of a PURL link -- arguably more important than title when it comes to citation, no? -- it should be explicitly visible in the resource, whether in SW and/or in the viewer wherever it may happen to be displayed.

**Comment by mejackreed** [comment]
@hannahfrost SearchWorks takes a advantage of a feature of embed, allowing it to hide the metadata panel. We moved the purl link to the metadata panel. The metadata panel looks like this:

<img width="583" alt="screen shot 2015-11-03 at 11 51 14 pm" src="https://cloud.githubusercontent.com/assets/1656824/10929785/006260d4-8286-11e5-8e4c-9e50237b4d84.png">

SW disables this panel, so the purl is not there. Seems like @jvine and I will need to discuss some more.

**Comment by andrewjbtw** [comment]
Closing this as embed shows a link to Purl in the info panel and the Stanford branding is present (including logo) in Mirador and the updated viewers. There are cases where it's still not that easy to find a purl link from a SW page (when there's a MARC record) but I don't think that's an embed issue.
